auth-api/src/auth_api/resources/task.py

```diff
@@ -23,27 +24,52 @@
 from auth_api.exceptions import BusinessException
 from auth_api.schemas import TaskSchema
 
-
 API = Namespace('tasks', description='Endpoints for tasks management')
 TRACER = Tracer.get_instance()
 _JWT = JWTWrapper.get_instance()
 
 
-@cors_preflight('GET,POST,OPTIONS')
-@API.route('', methods=['GET', 'POST', 'OPTIONS'])
+@cors_preflight('GET,OPTIONS')
+@API.route('/<string:task_relationship_type>', methods=['GET', 'POST', 'OPTIONS'])
```

Comment:
May be its better to get the tasks based on type rather than task_relationship_type, or make it as query param.